scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -47,12 +51,18 @@
   []
   (java.util.Date.))
 
+(defn offer-resource-value
+  ([offer resource-name]
+   (offer-resource-value offer resource-name :scalar))
+  ([offer resource-name value-type]
+   (->> offer :resources (filter #(= (:name %) resource-name)) first value-type)))
```

Comment:
Why are we taking the first? 

Suppose a single offer has "cpus" from both role "cook" and role "*", won't this mean we are throwing out one of them?